assets/src/edit-story/components/helpCenter/utils.js

```diff
@@ -16,7 +16,11 @@
 /**
  * Internal dependencies
  */
-import { FOCUSABLE_POPUP_CHILDREN_SELECTOR, POPUP_ID } from './constants';
+import { FOCUSABLE_POPUP_CHILDREN_SELECTOR, POPUP_ID, TIPS } from './constants';
+
+export function getUnreadTips(state) {
+  return Object.keys(TIPS).filter((tip) => !state.readTips[tip]) || [];
+}
```

Comment:
Instead of having this util, can't `unreadTipsCount` just be a part of of the state returned by `useHelpCenter`?

Reply:
yea it could. Kinda one half dozen of the other. this could just as easily be an effect:
```
export function deriveUnreadTipsCount(previous, next) {
  return Object.keys(TIPS).filter((tip) => !next.readTips[tip]).length || 0 ;
}
```

Reply:
maybe more inline with practices of existing setup 🤷

Reply:
CI shouldn't be a deciding factor for such changes I'd say :-) But no problem. I can fix it too in a related PR I'm working on.

Reply:
lol I'll tack it on right now